### DISCUSSION TITLE
Add parseMissingKeyHandler like in i18n react

### DISCUSSION THREAD

**Discussion by tokomeno-kaltura** [vinissimus/next-translate#989] [Ideas]
https://www.i18next.com/overview/configuration-options#missing-keys

Can't add parseMissingKeyHandler like I used to do with react-i18next

**Comment by aralroca** [comment]
did you try the `logger` from our configuration? https://github.com/aralroca/next-translate#3-configuration Is a function to log the missing keys in development and production
